scn/types/heartbeat.go

```diff
@@ -2,12 +2,12 @@ package types
 
 // HeartbeatRequest is a request to send heartbeat data.
 type HeartbeatRequest struct {
-	Version   string         `json:"version"`
-	Runnables *RunnableStats `json:"runnables"`
```

Comment:
What uses this?

Reply:
That's the payload that the SE2 API service receives when telemetry is sent. I don't think we can change this without it breaking telemetry.